common/utils.go

```diff
@@ -41,14 +42,18 @@ const HeaderWPATID = "X-WebPA-Transaction-Id"
 
 // TransactionLogging is used by the different Tr1d1um services to
 // keep track of incoming requests and their corresponding responses
-func TransactionLogging(reducedLoggingResponseCodes []int, logger kitlog.Logger) kithttp.ServerFinalizerFunc {
+func TransactionLogging(reducedLoggingResponseCodes []int, logger kitlog.Logger,headerConfig candlelight.HeaderConfig) kithttp.ServerFinalizerFunc {
 	errorLogger := logging.Error(logger)
+	spanIDHeaderName, traceIDHeaderName := candlelight.ExtractSpanIDAndTraceIDHeaderName(headerConfig)
 	return func(ctx context.Context, code int, r *http.Request) {
 		tid, _ := ctx.Value(ContextKeyRequestTID).(string)
 		transactionInfoLogger, ok := ctx.Value(ContextKeyTransactionInfoLogger).(kitlog.Logger)
+		traceId, spanId := candlelight.ExtractTraceInformation(r.Context())
+
+
 
 		if !ok {
-			errorLogger.Log(logging.MessageKey(), "transaction logger not found in context", "tid", tid)
+			errorLogger.Log(logging.MessageKey(), "transaction logger not found in context", "tid", tid,spanIDHeaderName, spanId,traceIDHeaderName,traceId)
```

Comment:
Was `gofmt` run on this code? Normally there are spaces between parameters.

Reply:
I missed it here. I will do it in next commit.